src/checkMatches.ts

```diff
@@ -119,18 +85,18 @@ const checkContextMatchesConditions = (filters: IFilter, context: object, option
   for (let [logicalOperator, isValidConditions] of Object.entries(returnValues)) {
     operatorsValues.push(logicalOperators[logicalOperator](isValidConditions));
   }
-  const isValid = and(operatorsValues);
+  const isValid = logicalOperators["AND"](operatorsValues);
```

Comment:
Pourquoi pas utiliser `import { and, not, or } from './logicalOperators';` comme dans conditionalOperators.ts ?